src/Module.Server/Modes/Conquest/CrpgConquestServer.cs

```diff
@@ -451,12 +463,12 @@ private void OnStageEnd()
         }
         else if (closestAgentToFlag != null)
         {
-            if (closestAgentToFlag.Team != flagOwner && !isContested)
+            if (flagCaptureWeight < 0 && !isContested)
             {
                 return CaptureTheFlagFlagDirection.Down;
             }
 
-            if (closestAgentToFlag.Team == flagOwner && isContested)
+            if (flagCaptureWeight > 0 && isContested)
             {
                 return CaptureTheFlagFlagDirection.Up;
             }

```

Comment:
The whole if structure from line 452to line 479 needs a refactor. It can now probably work with a single if + else

Reply:
its nearly the original TW code and work. If they make changes of flagsystem in the future that affect us too, its better to have similar code. Easier too find what we have to adjust in our script

Reply:
This reasoning works for HUD code where we copy paste vast ammount of TW code to reimplement to then make tiny changes. Gamemode are our own,. Takeo took the siege mode code as a base to make his own gamemode. 
The flagsystem logic of flags goes up and flag goes down is simple enough for us to not have to keep any relic of TW code.

Reply:
I'll refactor it myself :)

Reply:
Please do not open any further pull request. If you're not willing to participate in pull request review, contributing to an open source project is not for you. Especially with that attitude.